modules/ingest-common/src/test/java/org/opensearch/ingest/common/RemoveProcessorTests.java

```diff
@@ -67,12 +64,24 @@ public void testRemoveNonExistingField() throws Exception {
         config.put("field", fieldName);
         String processorTag = randomAlphaOfLength(10);
         Processor processor = new RemoveProcessor.Factory(TestTemplateService.instance()).create(null, processorTag, null, config);
-        try {
+        assertThrows("field [" + fieldName + "] doesn't exist", IllegalArgumentException.class, () -> {
             processor.execute(ingestDocument);
             fail("remove field should have failed");
```

Comment:
We probably don't need a `fail` inside the `assertThrows` (here or below). 

If the execute call doesn't throw an exception, it will already fail the assertion and therefore fail the test.